src/components/NumberInput/NumberInput.jsx

```diff
@@ -53,6 +53,10 @@ class NumberInput extends React.Component {
       requiredSpan = <span className="form-required-span">(*Required)</span>;
     }
 
+    const ariaDescribedby =
+      [errorSpanId, this.props.ariaDescribedby || ''].join(' ').trim() ||
```

Comment:
Why are we using `errorSpanId` here and not in the `Date` component?

Reply:
The `Date` component doesn't handle errors, it lets the `Select` and `NumberInput` components handle them.